plugin/loader/load_nocgo.go

```diff
@@ -0,0 +1,17 @@
+// +build linux,!cgo,!noplugin darwin,!cgo,!noplugin
```

Comment:
```suggestion
// +build !cgo,!noplugin
// +build linux darwin
```

nit: This might be a little easier to look at. Would also make the corresponding change in `plugin/loader/load_unix.go` since it makes it easier to visually verify that only one of `load_nocgo, load_noplugin, load_unix` actually are used.

Reply:
Ah, nice. That's much cleaner.